CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
  - Current development changes [ to be moved to release ]
+ - Export Device UDIDs when pre-booting simulators
+ - Added wait-until-simulator-booted command
```

Comment:
This can be removed as I will bump up the orb version when this PR is merged

Reply:
done